bootloader/bootloader.go

```diff
@@ -327,3 +327,16 @@ func removeKernelAssetsFromBootDir(bootDir string, s snap.PlaceInfo) error {
 
 	return nil
 }
+
+// ForGadget returns a bootloader matching a given gadget by inspecting the
+// contents of gadget directory or en error if no matching bootloader is found.
```

Comment:
I missed this in the previous PR but there's a typo here, s/en/an/

Reply:
Fixed already.